test/aggregate.test.js

```diff
@@ -1277,25 +1264,23 @@ describe('aggregate: ', function() {
       const mySchema = new Schema({ name: String, qty: Number });
       mySchema.index({ qty: -1, name: -1 });
       const M = db.model('Test', mySchema);
-      M.on('index', function(error) {
+      M.on('index', async function(error) {
```

Comment:
We should replace `M.on('index', async function(error) { ... })` with `await M.init(); ...`

Reply:
Ah yes, I've avoided modifying those tests.
`await M.init();` seems like the way to go. 👍